src/5e-SRD-Classes.json

```diff
@@ -9,76 +9,84 @@
         "type": "proficiencies",
         "from": [
           {
-            "url": "/api/proficiencies/skill-animal-handling",
-            "name": "Skill: Animal Handling"
+            "index": "skill-animal-handling",
+            "name": "Skill: Animal Handling",
```

Comment:
Not related to this PR, but I wonder if these should just have `Skill:` drop from their name and index.

Reply:
Probably a good shout